_pages/username.yaml

```diff
@@ -0,0 +1,3 @@
+githubHandle: tylergarrett
+pageUrl: tylergarrett.github.io
+timestamp: 2017-01-25
```

Comment:
@tylergarrett make sure you're following the exact syntax you see on https://services.github.com/on-demand/github-desktop/13/ (with the dashes above and below your code). Also, change the filename so instead of `username.yaml` it says `tylergarrett.yaml`.